packages/eva/mapping.json

```diff
@@ -388,16 +388,16 @@
         "filled": {
           "mapping": {
             "borderRadius": 4,
-            "borderWidth": 2,
+            "borderWidth": 1,
             "textColor": "text-control-color",
             "iconTintColor": "text-control-color",
             "iconMarginHorizontal": 4,
             "state": {
               "disabled": {
-                "borderColor": "border-basic-color-3",
-                "backgroundColor": "background-basic-color-3",
-                "textColor": "text-disabled-color",
-                "iconTintColor": "text-disabled-color"
+                "borderColor": "transparent",
```

Comment:
Wouldn't this change make the button visually smaller? I guess it should not be transparent, but somethig closer to `color-basic-600-transparent-24` that is set to `backgroundColor` (if we still have issues in React Native with it)

Reply:
Needs discussion